packages/devtools_app/lib/src/debugger/flutter/codeview.dart

```diff
@@ -219,11 +227,12 @@ class GutterRow extends StatelessWidget {
   Widget build(BuildContext context) {
     final theme = Theme.of(context);
 
-    final foregroundColor = theme.textTheme.bodyText2.color;
+    final foregroundColor =
```

Comment:
does the primaryColor look bad in the dark theme?

Reply:
It blends in with the editor's gutter color -